worker/src/main/java/org/verapdf/webapp/worker/listener/ExecutableTaskWorker.java

```diff
@@ -128,7 +128,7 @@ private File getFileToProcess(UUID fileId) throws VeraPDFWorkerException {
 
 	private StoredFileDTO saveReportFile(ValidationReport validationReport, UUID fileId)
 			throws VeraPDFWorkerException {
-		File reportFile = new File(workDir, "report-" + fileId);
+		File reportFile = new File(workDir, "report-" + fileId + ".json");
```

Comment:
This should not be done. The content type header should be set during the request from client. While we are not required to use some sort of extension for our files